lambda-functions/Lambda/SQLAlchemy/lambda_reddit_streaming.py

```diff
@@ -0,0 +1,49 @@
+import pandas as pd
+import praw
+from sqlalchemy import create_engine
+from db_utils import *
+
+engine = create_engine(db_host)
+
+REDDIT_CLIENT_ID = "VXYCrQAymaBrLQ"
+REDDIT_CLIENT_SECRET = "0Wwvv9zptchx7iqweK52cvbtX4A"
+REDDIT_USER_AGENT = "WSB"
+
+reddit = praw.Reddit(client_id=REDDIT_CLIENT_ID,
+                     client_secret=REDDIT_CLIENT_SECRET,
+                     user_agent=REDDIT_USER_AGENT)
+
+class TWICE(object):
+
+    def __init__(self, subreddits):
+
+        self.subreddits = subreddits
+
+    def subreddit_query(self, subreddits):
+
+        query = ""
```

Comment:
It looks like this function is trying to do the following, consider replacing it's body with:
```suggestion
       return "+".join(self.subreddits)
```